### ISSUE TITLE
Lack of support for positional arguments

### ISSUE THREAD

**Issue by MarioVilas** [arrai-innovations/argparse-color-formatter#9]
If I try to use colors in a positional variable metavar value, I get a broken output. Most likely this is because this library was originally written for Python 2.x which did not support positional arguments.

To reproduce the issue:

```
#!/usr/bin/python3

from argparse import ArgumentParser
from argparse_color_formatter import ColorHelpFormatter
from colorama import *

parser = ArgumentParser(formatter_class=ColorHelpFormatter)
parser.add_argument('withcolors', metavar=Fore.CYAN+"withcolors"+Style.RESET_ALL,
    help="this is an example long help text string to show the issue")
parser.add_argument('nocolors',
    help="this is an example long help text string to show the issue")
parser.print_help()
```

<img width="589" alt="Screenshot 2024-03-14 at 21 39 11" src="https://github.com/arrai-innovations/argparse-color-formatter/assets/227923/324b3f71-4015-4dec-bc34-331e51f2fbe5">

**Comment by jhillacre** [comment]
Thank you for bringing this issue to our attention.

1. Regarding the library being originally written for Python 2.x and its support for positional arguments, [I'd like to clarify that Python 2.x indeed supported positional arguments in `argparse`](https://python.readthedocs.io/en/v2.7.2/library/argparse.html#name-or-flags). The issue you're encountering is not related to the version of Python but rather to how the standard library deals with colour codes in metavar values for arguments.
2. As you've noticed, the problem stems from a need for colour awareness for argument names. We've implemented enhancements in pull request #11, which fixed a couple of areas not yet doing colour stripping before checking lengths and one spot doing string padding without regard for ANSI escapes. [Version 2.0.0](https://github.com/arrai-innovations/argparse-color-formatter/releases/tag/untagged-af2d57f08670b35be155) will be the released soon with this fix.
3. When we started work on the library again, we dusted off the infrastructure and dropped Python versions 2.7, 3.5, and 3.6. We hope this will be okay with you.

We appreciate your feedback and contributing to the improvement of `argparse-color-formatter`.
